diagnosis/getkuberneteslogs.sh

```diff
@@ -191,6 +165,27 @@ if [ $? -ne 0 ]; then
     exit 1
 fi
 
+#checks if azure-cli is installed   
+requirements
+
+#workaround for SSL interception
+export AZURE_CLI_DISABLE_CONNECTION_VERIFICATION=1 
+export ADAL_PYTHON_SSL_NO_VERIFY=1
```

Comment:
We may want to rollback this change after the log collector is done. See the `trap` command.